### PR TITLE
Timed Jump Actions 124-126

### PR DESCRIPTION
**Pull request by FS-21**
Action 124,0 : Stop the Timed jumps.

Action 125,x : When the timer reaches 0 the current action ends and jumps to the next Script Action line.

Action 126,x : When the timer reaches 0 the current action ends and restarts again the current Script Action line. Infinite loop timer.

These Actions are measured in game ticks, not seconds so is recommended big values.

### CODE REVIEW

**Review by Metadorius — CHANGES_REQUESTED** [review]
Doc/changelog/credits entries?

File: docs/New-or-Enhanced-Logics.md [hunk 1]
```diff
@@ -980,6 +980,37 @@ In `aimd.ini`:
...
+```
+
+### `125` Start a Timed Jump to the next line
+
+- When the timer ends the current script action ends and start the next one in the script type list.
+
+In `aimd.ini`:
+```ini
+[SOMESCRIPTTYPE]  ; ScriptType
+x=125,n           ; integer n=0
```
Comment (Metadorius) [inline docs/New-or-Enhanced-Logics.md:1001]: What does n=0 mean here? should be a bit more verbose.

File: docs/New-or-Enhanced-Logics.md [hunk 2]
```diff
@@ -980,6 +980,37 @@ In `aimd.ini`:
...
+```
+
+### `126` Start a Timed Jump to the same line
+
+- When the timer ends the current script action ends and start again the same script action. The timer jump repeats again (infinite loop) until is stopped with action 124 or the team is destroyed.
+
+In `aimd.ini`:
+```ini
+[SOMESCRIPTTYPE]  ; ScriptType
+x=126,n           ; integer n=0
```
Comment (Metadorius) [inline docs/New-or-Enhanced-Logics.md:1011]: Same here.

File: docs/New-or-Enhanced-Logics.md (hunk 1 above)
Comment (FS-21) [reply docs/New-or-Enhanced-Logics.md:1001]: TimerStruct what is? ticks/frames/seconds ... ? I don't know if writing "frames" is the right word :-/

Comment (Metadorius) [reply docs/New-or-Enhanced-Logics.md:1001]: Ingame measurment is done in frames usually.

**Comment by SMxReaver** [comment]
All three scripts appear to be functioning as intended here, everything looks good! 124 successfully kills the jump timer, 125 will jump forward, and 126 jumps backwards and continues the loop.

**Pull request merged**